coronavirus.md

```diff
@@ -75,7 +75,9 @@ There are three main types of compartmental models in epidemiology:
      Susceptible-Infected-Recovered (SIR), where each person can be  in one of three states, susceptible (S), infected (I) or recovered (R).
 In this model we study the number of people in each of those groups, denoted by variable S, I and R correspondingly. SIR system can be  expressed by the following set of ordinary differential equations: 
 
-$\frac{dS}{dt} = -\beta SI,$ 
```

Comment:
BTW we could enable mathjax which would allow you to add mathematical notation 😄